2024-09-06 10:40:38,961:DEBUG:certbot._internal.main:certbot version: 2.11.0
2024-09-06 10:40:38,962:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-09-06 10:40:38,962:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path', '/var/www/certbot/', '--dry-run', '-d', 'qina.sk']
2024-09-06 10:40:38,962:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-09-06 10:40:38,987:DEBUG:certbot._internal.log:Root logging level set at 30
2024-09-06 10:40:38,989:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-09-06 10:40:38,989:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7f758c9852b0>
Prep: True
2024-09-06 10:40:38,990:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7f758c9852b0> and installer None
2024-09-06 10:40:38,990:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-09-06 10:40:39,078:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-staging-v02.api.letsencrypt.org/acme/acct/161467813', new_authzr_uri=None, terms_of_service=None), ec4a1e905aaadf10e93ab6f8b67658ee, Meta(creation_dt=datetime.datetime(2024, 8, 30, 22, 36, 59, tzinfo=<UTC>), creation_host='96d83655557f', register_to_eff=None))>
2024-09-06 10:40:39,079:DEBUG:acme.client:Sending GET request to https://acme-staging-v02.api.letsencrypt.org/directory.
2024-09-06 10:40:39,083:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-staging-v02.api.letsencrypt.org:443
2024-09-06 10:40:39,249:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 820
2024-09-06 10:40:39,250:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 06 Sep 2024 10:40:39 GMT
Content-Type: application/json
Content-Length: 820
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "D7CsRhI7Jfg": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-staging-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org/docs/staging-environment/"
  },
  "newAccount": "https://acme-staging-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-staging-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-staging-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-staging-v02.api.letsencrypt.org/acme/revoke-cert"
}
2024-09-06 10:40:39,251:DEBUG:certbot._internal.display.obj:Notifying user: Simulating a certificate request for qina.sk
2024-09-06 10:40:39,267:DEBUG:acme.client:Requesting fresh nonce
2024-09-06 10:40:39,267:DEBUG:acme.client:Sending HEAD request to https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce.
2024-09-06 10:40:39,320:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-09-06 10:40:39,321:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 06 Sep 2024 10:40:39 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 6wrlOJ0TJyOHbasN6BEET0_uArLgnlLU8wzP1jpEu-nLKP6tBvs
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-09-06 10:40:39,321:DEBUG:acme.client:Storing nonce: 6wrlOJ0TJyOHbasN6BEET0_uArLgnlLU8wzP1jpEu-nLKP6tBvs
2024-09-06 10:40:39,322:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "qina.sk"\n    }\n  ]\n}'
2024-09-06 10:40:39,326:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xNjE0Njc4MTMiLCAibm9uY2UiOiAiNndybE9KMFRKeU9IYmFzTjZCRUVUMF91QXJMZ25sTFU4d3pQMWpwRXUtbkxLUDZ0QnZzIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "pMf2solBe68ytW1hUOTX130uNl9DBN9EHlWRJeSdECGh_s8PhbcFuYNrw1njXgzqq78x21h12xYWgrTn87aDqaNVi5zGd_0jE_O_51vZPkEu54EE6oCBh0z1pF8txWO8XL4MmDd6l5EPvaFMgAzaJCeNlGLNx2vN-dsFh09iS8jt0NIrIsOfK3XUqgdAYG3PwFSwouMrEuli5HM4DODOyEGfPndFkfrbChnXbpCZQWiqQr0HATzrbjtTkKpzQ0dxKk2S9vrfmY0zqO5jgABFhIVhqnuiFdE_iGh_BF13D2hLVtWPmsclyAZ3JRN_0HKcX0rfxxsBhti3Hl3h1Y7zPA",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInFpbmEuc2siCiAgICB9CiAgXQp9"
}
2024-09-06 10:40:39,472:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 346
2024-09-06 10:40:39,472:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Fri, 06 Sep 2024 10:40:39 GMT
Content-Type: application/json
Content-Length: 346
Connection: keep-alive
Boulder-Requester: 161467813
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/order/161467813/18912711843
Replay-Nonce: 6wrlOJ0TWfDhvt9EENTv6k4srMz7Rp1CBUlWYqAwVx4RKJNBmrc
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-09-13T10:40:39Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "qina.sk"
    }
  ],
  "authorizations": [
    "https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/13893766743"
  ],
  "finalize": "https://acme-staging-v02.api.letsencrypt.org/acme/finalize/161467813/18912711843"
}
2024-09-06 10:40:39,473:DEBUG:acme.client:Storing nonce: 6wrlOJ0TWfDhvt9EENTv6k4srMz7Rp1CBUlWYqAwVx4RKJNBmrc
2024-09-06 10:40:39,473:DEBUG:acme.client:JWS payload:
b''
2024-09-06 10:40:39,476:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/13893766743:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xNjE0Njc4MTMiLCAibm9uY2UiOiAiNndybE9KMFRXZkRodnQ5RUVOVHY2azRzck16N1JwMUNCVWxXWXFBd1Z4NFJLSk5CbXJjIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzEzODkzNzY2NzQzIn0",
  "signature": "j1Fi0xQnGJMtjYbmThuWXa1XeIvTaRheym5J_ChsY4qvmLRAJGatbKdyckuE_9dkGuZrW05bw9OQNvyVtdu2w5KRiUo1uDOxKf1TSIFcGylQfR6mP511GqWpofkZAt8r2nskkHh2Y4xdxz5lUnqQUrXVXT5F9VaB8wgzaBe5o-C0Gh5uAmYvKVpN0R8oYbrt9kIff3N4SEQRnWYFLazsWcdr_VDykReuMEMKOZGFYhfAhDWzLS3Euzbw4EgZzeleXVrs5assJZkgK-FM0vU7lFTN_ErjhkB7KfolNGVRlskD6-2-Q9HxXLXofoR6g6e3VdFRR90TPl_e1zR-dDI-EA",
  "payload": ""
}
2024-09-06 10:40:39,535:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/13893766743 HTTP/1.1" 200 812
2024-09-06 10:40:39,536:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 06 Sep 2024 10:40:39 GMT
Content-Type: application/json
Content-Length: 812
Connection: keep-alive
Boulder-Requester: 161467813
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 6wrlOJ0TopDcvXscWR4R6fYNATeiHmuO_dd8j4_ORcfXFumx7XM
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "qina.sk"
  },
  "status": "pending",
  "expires": "2024-09-13T10:40:39Z",
  "challenges": [
    {
      "type": "dns-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/13893766743/aGCqqw",
      "status": "pending",
      "token": "dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8"
    },
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/13893766743/w7nGow",
      "status": "pending",
      "token": "dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/13893766743/ZoiwHw",
      "status": "pending",
      "token": "dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8"
    }
  ]
}
2024-09-06 10:40:39,537:DEBUG:acme.client:Storing nonce: 6wrlOJ0TopDcvXscWR4R6fYNATeiHmuO_dd8j4_ORcfXFumx7XM
2024-09-06 10:40:39,537:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-09-06 10:40:39,538:INFO:certbot._internal.auth_handler:http-01 challenge for qina.sk
2024-09-06 10:40:39,538:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2024-09-06 10:40:39,538:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2024-09-06 10:40:39,540:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8
2024-09-06 10:40:39,541:DEBUG:acme.client:JWS payload:
b'{}'
2024-09-06 10:40:39,544:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/13893766743/w7nGow:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xNjE0Njc4MTMiLCAibm9uY2UiOiAiNndybE9KMFRvcERjdlhzY1dSNFI2ZllOQVRlaUhtdU9fZGQ4ajRfT1JjZlhGdW14N1hNIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLXYzLzEzODkzNzY2NzQzL3c3bkdvdyJ9",
  "signature": "BJTXHATf2_7JOzLImLwOG2XEhH6h0FkhtcnV9TobU_JPjJvjTqfnz40wR6s2qxDsL2e8wlW1oPee-Fafwt4bCY0Leo2iBOARIpgx5mfADzKdo8djIFNuo9O0QGabdC52raRBSjzK48BhxQtHCAB0AgKhoFJzfC3WXUrVcsMjXePIm7D6qzM-sTML0QAFuV9pwFWuyjgTaBsr6wo4IBjdMx1dCHWZHGoZgwuHfZdxMz0ygBh64aFORMYVCq9jWzwhrzZ4bmoBYv1uCWxg9z4fE5zj1Efpw_fg6BEzhW6uUKKh3l3lN1j8IedQrLu6aSu_9jh7Fd69EP3hmpsYMltoqQ",
  "payload": "e30"
}
2024-09-06 10:40:39,608:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/13893766743/w7nGow HTTP/1.1" 200 194
2024-09-06 10:40:39,609:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 06 Sep 2024 10:40:39 GMT
Content-Type: application/json
Content-Length: 194
Connection: keep-alive
Boulder-Requester: 161467813
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/13893766743>;rel="up"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/13893766743/w7nGow
Replay-Nonce: 6wrlOJ0TMzgL_yBxDHneZ1N6HOYDSx8vviA2n8ug0boBs_bRF5E
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/13893766743/w7nGow",
  "status": "pending",
  "token": "dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8"
}
2024-09-06 10:40:39,609:DEBUG:acme.client:Storing nonce: 6wrlOJ0TMzgL_yBxDHneZ1N6HOYDSx8vviA2n8ug0boBs_bRF5E
2024-09-06 10:40:39,610:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-09-06 10:40:40,610:DEBUG:acme.client:JWS payload:
b''
2024-09-06 10:40:40,613:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/13893766743:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xNjE0Njc4MTMiLCAibm9uY2UiOiAiNndybE9KMFRNemdMX3lCeERIbmVaMU42SE9ZRFN4OHZ2aUEybjh1ZzBib0JzX2JSRjVFIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzEzODkzNzY2NzQzIn0",
  "signature": "SgDXsNzK8kJ5YsmA03r4PEAv7UXu0oJEIp8990nXZB8M1n1fqxPg8qqyvzjlUxe3po07kRMbD5XQqeQSYMcFZdvY2GvW0xER2zmyEFq5Y_-fWdvO80SRvChDaR9b8fW2MkfytFnkKE47JPZSTybzFWyA6XkVpl8XZ6enJhRyyUwwo8Qm-thiw2grgIkUV8piAmuLWmLVERcgIDxvMb8WooA4Up_uHeYJ7t6Y0oCeH7_2BdpNvbE0ZoqTV8sFLZKIFLv2CsHU6rQ7ddJ7bkAJ7aVwPm9bP3KpR8NIc-qA9pGiZInS8tgGH4Dg757cuDCSnZZcZ125pBFFJz4ETQTSXw",
  "payload": ""
}
2024-09-06 10:40:40,671:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/13893766743 HTTP/1.1" 200 1061
2024-09-06 10:40:40,672:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 06 Sep 2024 10:40:40 GMT
Content-Type: application/json
Content-Length: 1061
Connection: keep-alive
Boulder-Requester: 161467813
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: _yLDrU7H6lBH9chSHRBPhoZp8NQENIAwASiIj07Ig7wtkXNvFCc
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "qina.sk"
  },
  "status": "invalid",
  "expires": "2024-09-13T10:40:39Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall-v3/13893766743/w7nGow",
      "status": "invalid",
      "validated": "2024-09-06T10:40:39Z",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "2a00:4b40:aaaa:2009::6: Invalid response from http://qina.sk/.well-known/acme-challenge/dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8: 404",
        "status": 403
      },
      "token": "dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8",
      "validationRecord": [
        {
          "url": "http://qina.sk/.well-known/acme-challenge/dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8",
          "hostname": "qina.sk",
          "port": "80",
          "addressesResolved": [
            "159.223.188.113",
            "2a00:4b40:aaaa:2009::6"
          ],
          "addressUsed": "2a00:4b40:aaaa:2009::6"
        }
      ]
    }
  ]
}
2024-09-06 10:40:40,672:DEBUG:acme.client:Storing nonce: _yLDrU7H6lBH9chSHRBPhoZp8NQENIAwASiIj07Ig7wtkXNvFCc
2024-09-06 10:40:40,673:INFO:certbot._internal.auth_handler:Challenge failed for domain qina.sk
2024-09-06 10:40:40,673:INFO:certbot._internal.auth_handler:http-01 challenge for qina.sk
2024-09-06 10:40:40,673:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: qina.sk
  Type:   unauthorized
  Detail: 2a00:4b40:aaaa:2009::6: Invalid response from http://qina.sk/.well-known/acme-challenge/dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8: 404

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-09-06 10:40:40,675:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-09-06 10:40:40,676:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-09-06 10:40:40,676:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-09-06 10:40:40,676:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/dmLnU622lv5lrCov_sDHm4CRXi_rJju0zdylxQZGhZ8
2024-09-06 10:40:40,676:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-09-06 10:40:40,677:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1894, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-09-06 10:40:40,682:ERROR:certbot._internal.log:Some challenges have failed.
